public/app/features/scenes/models/types.ts

```diff
@@ -57,11 +62,17 @@ export interface SceneObject<TState extends SceneObjectState = SceneObjectState>
   /** Called when component unmounts. Unsubscribe to events */
   onUnmount(): void;
 
+  //** Get the scene editor */
+  getSceneEditor(): SceneEditor;
+
```

Comment:
if we had an event system we could get rid of this, the Main reason it's here is for SceneComponentEditingWrapper to get the SceneEditor to update hoverObject or selectedObject